src/main/java/io/github/fabricators_of_create/porting_lib/transfer/item/ItemStorageHandler.java

```diff
@@ -17,17 +18,21 @@
 
 	public ItemStorageHandler(Storage<ItemVariant> storage) {
 		this.storage = storage;
+		getSlots();
 	}
-
+	private int slotCount = 0;
```

Comment:
slot count can *not* be assumed to be stable, Create breaks this on it's own with deployers.